src/main/java/mate/academy/spring/util/DataInitializer.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.util;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+
+public class DataInitializer {
```

Comment:
As an alternative you can mark this class as a @Component. Then there would be no need to create additional method in AppConfig